notebooks/00_quick_start/lightgbm_tinycriteo.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>Remove: 'After data preparation' as this now preceeds that section</p><p><br></p><p>This is predicting click or no click, might be helpful to state that directly for clarity.</p><p><br></p><p>Also, helpful to clarify that Area Under the Curve (AUC) is used as a metric which is less effected by imbalance in the classes of the dataset.</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
fixed, thanks.